jams/schemata/namespaces/pitch/note_hz.json

```diff
@@ -0,0 +1,8 @@
+{"note_hz":
+    {
+        "value": { "type": "number" },
```

Comment:
Looks like this should be
```"value": { "type": "number", "minimum": 0},```

because the following test is failing:
```namespace_tests.test_ns_note_hz_invalid(-23,) ... FAIL```